### ISSUE TITLE
Empty "Trust OMEMO Fingerprints" screen after generating keys in ARC-welded Conversations

### ISSUE THREAD

**Issue by magicvodi** [iNPUTmice/Conversations#1476]
As mentioned by me and confirmed by @iNPUTmice ( https://github.com/siacs/Conversations/issues/1246#issuecomment-146834637 ) OMEMO keys which are generated in ARC are not readable by other clients. Thus I and my contacts get an empty "Trust OMEMO Fingreprints" screen before sending any message. And it comes for every single message.

Is there any way to get rid of this screen? By deleting the ARC-generated keys? By manually uploading(?) them?

**Comment by iNPUTmice** [comment]
> Is there any way to get rid of this screen? By deleting the ARC-generated keys? By manually uploading(?) them?

If your ARC-Conversations is no longer active you can select 'Clear devices' from the overflow menu of Account details on one of your other devices.

**Comment by magicvodi** [comment]
Thank you very much, that helped!

**Comment by iNPUTmice** [comment]
The issue remains in that the Trust Keys Activity should not (repeatably) show up  if there are broken keys in someones PEP @strb

**Comment by RalfJung** [comment]
I also just saw an empty "Confirm Keys" while playing around with the Gajim OMEMO plugin: A friend had a new OMEMO key generated by his Gajim. The next time I sent a message to him, I got an empty window asking me to confirm a key. A few minutes later, after he sent his first OMEMO message from Gajim to me, I saw the red lock and then, finally, when I sent the next message, I could confirm his key.

The least that could be done is not to even open the Confirm Keys window if it will be shown empty anyway.
